src/sandbox/sandbox_state.js

```diff
@@ -0,0 +1,45 @@
+import { PricingStrategy } from '../utility/PricingStrategy';
+
+const service = {
+  service_row_id: 1,
+  org_id: 'snet',
```

Comment:
using double quotes everywhere.. you could try `npm run lint:fix' for the common issues.

Reply:
Same as [comment](https://github.com/singnet/refactored-dapp/pull/110#discussion_r309244829)